src/mailtemplates/scheduled-execution-failed/scheduled-execution-failed.html.mustache

```diff
@@ -0,0 +1,522 @@
+                    <tr>
+                        <td align="center" valign="top">
+                            <!-- BEGIN PREHEADER // -->
+                            <table border="0" cellpadding="0" cellspacing="0" width="100%" id="templatePreheader">
+                                <tr>
+                                    <td valign="top" class="preheaderContent" style="padding-top:10px; padding-right:20px; padding-bottom:10px; padding-left:20px;" mc:edit="preheader_content00">
+                                        Scheduled changes execution failed
+                                    </td>
+                                    <!-- *|IFNOT:ARCHIVE_PAGE|* -->
+                                    <td valign="top" width="180" class="preheaderContent" style="padding-top:10px; padding-right:20px; padding-bottom:10px; padding-left:0;" mc:edit="preheader_content01">
+                                        Email not displaying correctly?<br /><a href="*|ARCHIVE|*" target="_blank">View it in your browser</a>.
+                                    </td>
+                                    <!-- *|END:IF|* -->
+                                </tr>
+                            </table>
+                            <!-- // END PREHEADER -->
+                        </td>
+                    </tr>
+                    <tr>
+                        <td align="center" valign="top">
+                            <!-- BEGIN HEADER // -->
+                            <table border="0" cellpadding="0" cellspacing="0" width="100%" id="templateHeader">
+                                <tr>
+                                    <td valign="top" class="headerContent">
+                                        <img src="https://cdn.getunleash.io/unleash_logo_600.png" style="max-width:600px;padding:1rem;" id="headerImage" mc:label="header_image" mc:edit="header_image" mc:allowdesigner mc:allowtext />
+                                    </td>
+                                </tr>
+                            </table>
+                            <!-- // END HEADER -->
+                        </td>
+                    </tr>
+                    <tr>
+                        <td align="center" valign="top">
+                            <!-- BEGIN BODY // -->
+                            <table border="0" cellpadding="0" cellspacing="0" width="100%" id="templateBody">
+                                <tr>
+                                    <td valign="top" class="bodyContent" mc:edit="body_content">
+                                        <h1>Scheduled changes execution failed</h1>
+
+                                        <br />
+                                        Scheduled change request <span><a class="changeRequestLink" href="{{{ changeRequestLink }}}" target="_blank" rel="noopener noreferrer">{{{ changeRequestLink }}}</a></span> failed to apply at {{{ scheduledAt }}} due to {{{ errorMessage }}}.
```

Comment:
Does this use the link to the change request as the title? As in https://hosted.unleash.io/bob/projects/x/change-requests/9? That feels confusing. Can we use the change request id and title (if present)?

Reply:
Changed: if title is present would be `id: title` if not `id`